libafl_cc/src/lib.rs

```diff
@@ -76,6 +78,94 @@ pub enum Error {
     Unknown(String),
 }
 
+/// `LibAFL` target configuration
+#[derive(Debug, Clone)]
+pub enum Configuration {
```

Comment:
Would it make sense to have these be flags, with the option to `or` these? Like, I'd want to use UBsan and ASan in the same node sometimes(?)

Reply:
Yes. I mentioned that in the feature description in the OP. I didn't do it yet, but it should be possible.

Reply:
I added the `Configuration::Compound` variant, which takes a vector of other `Configuration`s, allowing for compound configurations.